dist/ShareButton.js

```diff
@@ -2086,6 +1921,6 @@ String.prototype.capFLetter = function () {
 exports["default"] = ShareUtils;
 module.exports = exports["default"];
 
-},{}]},{},[22])
-(22)
+},{}]},{},[36])
+(36)
 });
```

Comment:
yeah, postprocessing would work... it might be better to just fix whatever tool produced this file, but as a workaround, you can just add the final newline with a separate command